src/main/java/hudson/plugins/git/GitPublisher.java

```diff
@@ -573,15 +573,15 @@ public String getDisplayName() {
         private String noteNamespace;
         private boolean noteReplace;
 
-        public String getnoteMsg() {
+        public String getNoteMsg() {
```

Comment:
I originally changed this as i was getting:
```
WARNING: failed to uncoerce hudson.plugins.git.GitPublisher$NoteToPush@12647296
java.lang.UnsupportedOperationException: no public field ‘noteMsg’ (or getter method) found in class hudson.plugins.git.GitPublisher$NoteToPush
```

When using the snippet generator, which resulted in this snippet being generated.
```
gitPublisher notesToPush: [<object of type hudson.plugins.git.GitPublisher.NoteToPush>], url: 'git@github.com:iwarapter/git-plugin.git'
```

Interestingly the a normal freestyle job doesn't suffer from this and the jelly config is pretty much the same. Any thoughts?